src/main/java/com/github/freeclimbapi/utils/RequestVerifier.java

```diff
@@ -0,0 +1,70 @@
+package com.github.freeclimbapi.utils;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Date;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import org.apache.commons.codec.binary.Hex;
+
+public class RequestVerifier {
+
+    public void verifyRequestSignature(String requestBody, String requestHeader, String signingSecret,
+            int tolerance) {
```

Comment:
Requirements specify a default value for `tolerance`.
General java conventions usually prefer `Integer` over `int` - but could be swayed depending on the case

Reply:
I can change this to be Integer, I used int because that was the convention I used to use for numbers but I can change that in the latest commit